app-emulation/rkt/rkt-9999.ebuild

```diff
@@ -32,7 +32,7 @@ DEPEND=">=dev-lang/go-1.2
 RDEPEND=""
```

Comment:
The SDK currently uses go-1.3.3. This is just forcing the version to be _at least_ go-1.2.